starter-code/scripts/controllers/articleController.js

```diff
@@ -13,16 +13,19 @@
     }
   };
 
-  // COMMENT: What does this method do?  What is it's execution path?
+  // COMMENTed: What does this method do?  What is it's execution path?
+  // This method redirects the page to load the remaining portion of the article upon clicking on read-on. Its execution path would be upon click of read on it would look through all instances of the artciles, locate the article ID you selected, findwhere the article ID is the same as the selected article and then calls the next function articleControll.index which passes the article content into the articleView.index as an arguement, if an article ID was not found it will reload the home page.
```

Comment:
Great! This method is triggered as the first callback upon the client hitting the '/article/:id' route (when a user clicks on "read on"). This captures the id of the article associated with that Read On click, and that id now lives on the ctx object as ctx.params.id.  The method calls Article.findWhere, passing it the 'id' field, the value of that id (ctx.params.id), and a callback function.  Article.findWhere executes a database query that finds the article with that ID in the database, and then, Article.findWhere calls the callback function (articleData).  articleData attaches the found article from the database to the ctx object (to reference later) and calls the next method in the route (articleController.index).